README.md

```diff
@@ -2,4 +2,45 @@
 
 A simple example showing how to setup a 3d scene across windows on the same origin using three.js and localStorage. Code should be self explanatory
 
-follow me on twitter at [@_nonfigurativ_](https://twitter.com/_nonfigurativ_) for more stuff like this
\ No newline at end of file
+follow me on twitter at [@_nonfigurativ_](https://twitter.com/_nonfigurativ_) for more stuff like this
+
+
+
+## HOW TO RUN THE PROJECT
+
+#### 1) Install http-server globaly:
+-------------
+
+`sudo npm install http-server -g`
```

Comment:
why using `sudo` ? You can install "globally" for the user
ref:
* https://docs.npmjs.com/downloading-and-installing-packages-globally
* https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally (to create a $USER global cache)

Reply:
Or use npx?